code/Modules/Props/Types/Prop.cs

```diff
@@ -11,12 +11,15 @@ namespace ssl.Modules.Props.Types
     /// </summary>
     public partial class Prop : ModelEntity, ISelectable
     {
+        public const string Tag = "Prop";
+        
```

Comment:
suggestion: place Tag in a upper level.
If Item and props have tag, they have to be in an upper level